tms/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
```

Comment:
This file is necessary ?

Reply:
@marcelsavegnago This is mandatory since v17 on OCA. In fact, this is added automatically when running pre-commit. This replaces the setup/setuptools mechanisms to install addons as python packages.

Reply:
> @marcelsavegnago This is mandatory since v17 on OCA. In fact, this is added automatically when running pre-commit. This replaces the setup/setuptools mechanisms to install addons as python packages.

Thanks @rousseldenis. I haven't started doing anything on 17 yet. Thanks for the information.